src/scripts/main.js

```diff
@@ -354,7 +354,40 @@
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+'use strict';
```

Comment:
The 'use strict' directive is used twice in this file. It's only necessary to declare it once at the top of your JavaScript file. This directive helps you to write safer JavaScript by turning silent errors into throw errors.